docs/AboutUs.adoc

```diff
@@ -4,53 +4,53 @@
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
-_{The dummy content given below serves as a placeholder to be used by future forks of the project.}_ +
-{empty} +
+Meditabs was developed by the https://github.com/cs2103-ay1819s2-t12-3/main[t12-3] team. +
+
 We are a team based in the http://www.comp.nus.edu.sg[School of Computing, National University of Singapore].
 
 == Project Team
 
-=== Jonathan 
-image::Jonathan.jpg[width="150", align="left"]
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
+=== Lee Wei Hao Jonathan
+image::Jonathan.png[width="150", align="left"]
+{empty} [https://github.com/JonathanLeeWH[github]] [<<johndoe#, portfolio>>]
 
-Role: Project Advisor
+Role: Team Lead +
+      Responsibilities: UI
 
 '''
 
-=== John Roe
-image::lejolly.jpg[width="150", align="left"]
-{empty}[http://github.com/lejolly[github]] [<<johndoe#, portfolio>>]
+=== Tan Kian Wei, Jason
+image::Jason.png[width="150", align="left"]
+{empty}[https://github.com/jtankw3[github]] [<<johndoe#, portfolio>>]
 
-Role: Team Lead +
+Role: Developer +
 Responsibilities: UI
 
 '''
 
-=== Johnny Doe
+=== Ong Ai Hui
 image::yijinl.jpg[width="150", align="left"]
-{empty}[http://github.com/yijinl[github]] [<<johndoe#, portfolio>>]
+{empty}[https://github.com/flxffy[github]] [<<johndoe#, portfolio>>]
 
 Role: Developer +
-Responsibilities: Data
+Responsibilities: Storage
 
 '''
 
-=== Johnny Roe
+=== Law Yong Khan, Kevin
 image::m133225.jpg[width="150", align="left"]
 {empty}[http://github.com/m133225[github]] [<<johndoe#, portfolio>>]
 
 Role: Developer +
-Responsibilities: Dev Ops + Threading
+Responsibilities: Model
 
 '''
 
-=== Benson Meier
-image::yl_coder.jpg[width="150", align="left"]
-{empty}[http://github.com/yl-coder[github]] [<<johndoe#, portfolio>>]
+=== Melvin Vito
+image::Melvin.png[width="150", align="left"]
```

Comment:
Similar to the previous review:

Based on the new instructions in Week 6 CS2103T module website it showed that:
![image](https://user-images.githubusercontent.com/46712102/52902203-d8551700-3248-11e9-822a-2ad6d2afff8f.png)

The filename of each of the team members photo should be our Github username in lowercase so I think you might need to rename our .png image files to our Github username in lowercase.